pkgs/development/libraries/wayland/protocols.nix

```diff
@@ -6,14 +6,14 @@
 
 stdenv.mkDerivation rec {
   pname = "wayland-protocols";
-  version = "1.31";
+  version = "1.32";
 
   # https://gitlab.freedesktop.org/wayland/wayland-protocols/-/issues/48
   doCheck = stdenv.hostPlatform == stdenv.buildPlatform && stdenv.targetPlatform.linker == "bfd" && wayland.withLibraries;
 
   src = fetchurl {
     url = "https://gitlab.freedesktop.org/wayland/${pname}/-/releases/${version}/downloads/${pname}-${version}.tar.xz";
-    hash = "sha256-oH+nIu2HZ27AINhncUvJovJMRk2nORLzlwbu71IZ4jg=";
+    hash = "sha256-dFl5nTQMgpa2le+FfAfd7yTFoJsJq2p097kmQNKxuhE=";
   };
```

Comment:
Why not use `fetchFromGitLab`?
```nix
{ fetchFromGitLab }: 
{
	src = fetchFromGitLab {
    	domain = "gitlab.freedesktop.org";
    	owner = "wayland";
    	repo = "wayland-protocols";
    	rev = version;
    	hash = "sha256-ljMcY59iOdGYeToadCuQ3t0lrBnsBdot+NiT6e+JJ04=";
  	};
 }
```

Reply:
Because it's a dedicated release tarball asset, not the standard source code archive that `fetchFromGitLab` fetches.